docs/developer-docs/multi-chain/overview.mdx

```diff
@@ -0,0 +1,115 @@
+---
+keywords: [advanced, bitcoin, btc, ckbtc, overview, multichain, multi-chain, ethereum, evm, eth, cketh]
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Overview
+
+<MarkdownChipRow labels={["Advanced", "Multi-chain", "Concept" ]} />
+
+## What is multi-chain?
+
+The term 'multi-chain' refers to the Internet Computer's ability to trustlessly make direct calls to another blockchain, enabling dapps to expand their capabilities across multiple networks. ICP smart contracts can interact with another chain's smart contracts and process the chain's native assets. This ability to communicate directly with other blockchain networks is possible using the protocol's unique cryptography features, which support signing transactions that execute on other networks. ICP currently has multi-chain integrations with the Bitcoin and Ethereum networks.
```

Comment:
> and process the chain's native assets

I'm not a native speaker, but "process assets" sounds weird to me. WDYT about "manage ... assets"?